apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -218,6 +218,13 @@
             <version>${datasource-proxy.version}</version>
         </dependency>
 
+        <dependency>
+            <!-- Used for endpoint testing automation -->
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
👋 @StewGoin @keithdadkins 
Obviously, this is a new dependency scoped to **test** and will not be shipped alongside the BFD intermediate artifacts (JARs, WARs). But, here are some other highlights:
* Widely used.
  * \>150,000 _dependent_ projects in GitHub acording to [the project's dependency graph for io.rest-assured:rest-assured](https://github.com/rest-assured/rest-assured/network/dependents?package_id=UGFja2FnZS0yMDA3NTY0MDU%3D)
  *  \>2300 dependent artifacts according to [maven central](https://mvnrepository.com/artifact/io.rest-assured/rest-assured)
* Well maintained. [>130 contributors, >1,800 forks, >6,500 stars, and recent commit to trunk ~60 days ago](https://github.com/rest-assured/rest-assured).
* Active [StackOverflow Community surrounding `[rest-assured]`](https://stackoverflow.com/questions/tagged/rest-assured). >2200 questions for all time and 14 questions in August 2023 as of this writing.
* [_Very_ limited existing use within CMSgov](https://github.com/search?q=org%3ACMSgov%20rest-assured&type=code).

Reply:
I appreciate the additional notes here, it does make the security research a bit simpler :)